google/cloud/bigtable/row_set.py

```diff
@@ -109,6 +109,24 @@ def add_row_range_from_keys(
         row_range = RowRange(start_key, end_key, start_inclusive, end_inclusive)
         self.row_ranges.append(row_range)
 
+    def add_row_range_by_prefix_from_keys(self, row_key_prefix):
+        """Add row range to row_ranges list that start with the row_key_prefix from the row keys
+
+        For example:
+
+        .. literalinclude:: snippets_table.py
+            :start-after: [START bigtable_row_range_by_prefix_from_keys]
+            :end-before: [END bigtable_row_range_by_prefix_from_keys]
+
+        :type row_key_prefix: str
+        :param row_key_prefix: Add all rows that start with this row key prefix.
+                            Prefix cannot be zero length."""
```

Comment:
Incorrect indentation, and the comment doesn't seem to be very understandable.

Reply:
I don't think so it's indentation problem as you can see below
https://github.com/q-logic/python-bigtable/blob/1d92cc2e3c6dad83e505438b2e0ebdf0a483fbdf/google/cloud/bigtable/row_set.py#L101-L107